tools/RAiDER/cli/raiderGNSS_dl.py

```diff
@@ -0,0 +1,168 @@
+import argparse
```

Comment:
Here's where I think the unified CLI helps us a lot. We use the same CLI for everything, and then just call the step needed, similar to what you have working now.

Reply:
Ah, I did not mean to add this file into the push. It actually does nothing. I incorporated the dlGNSS step into the runProgram.py / raiderDelay step which are in effect the start of a unified CLI